README.md

```
@@ -47,3 +42,8 @@ Please feel free to send a pull request for any other free and downloadable top
 
 - Download page: https://radar.cloudflare.com/domains
 - Download link: https://radar.cloudflare.com/charts/LargerTopDomainsTable/attachment?id=1257&top=1000000
+
+### CrUX
+
+- Download page: https://github.com/zakird/crux-top-lists
```

Markdownlint (0.35.0)

Bare URL used

(MD034, no-bare-urls)